src/Components/Address/AddressFormFields.tsx

```diff
@@ -1,11 +1,34 @@
 import { ContextModule } from "@artsy/cohesion"
 import { Column, GridColumns, Input } from "@artsy/palette"
-import { useFormContext } from "Apps/Auction/Hooks/useFormContext"
 import { AddressAutocompleteInput } from "Components/Address/AddressAutocompleteInput"
+import {
+  Address,
+  basicPhoneValidator,
+  yupAddressValidator,
+} from "Components/Address/utils"
 import { CountrySelect } from "Components/CountrySelect"
+import { useFormikContext } from "formik"
 import { useAnalyticsContext } from "System/Hooks/useAnalyticsContext"
 
-export const AddressForm = () => {
+interface FormikContextWithAddress {
+  address: Address
+  phoneNumber?: string
+}
+
+interface Props {
+  withPhoneNumber?: boolean
+}
+
+export const addressFormFieldsValidator = (args: {
+  withPhoneNumber: boolean
```

Comment:
Should this be typed with `Props` right up above?

Reply:
Done.